AssignmentOne.java

```diff
@@ -0,0 +1,29 @@
+import java.util.HashMap;
+import java.util.LinkedList;
+
+public class AssignmentOne {
+
+	public static void main(String[] args) {
+				
+		SingleLinkedList<Integer> list = new SingleLinkedList<>(); 
+		
+		//The linked list is empty
+		try {
+			list.kToLast(2);
+		} catch (IllegalArgumentException e) {
+			System.out.println(e.getMessage());
+		}
+		
+	    // The final linked list is 5->4->3->2->1
```

Comment:
This type of linked list is called a Stack. Elements are added to the front of the list. Usually a linked list is a Queue where elements are added to the end of the list.